lang/en/docs/cli/actions/create-python-env.md

```diff
@@ -45,6 +47,32 @@ In order to disable the virtual environment and return to the original default c
 deactivate
 ```
 
+To reactivate an existing virtual environment, you just have to source the activate script again:
```

Comment:
It might be debatable whether this is the best practice or not, but we have purposefully employed the use of third-person when referring to the User (https://github.com/Exabyte-io/documentation/blob/master/WRITING-STYLE.md, see "Use third person") before for the rest of the documentation, and should be consistent here too.

Reply:
addressed in https://github.com/Exabyte-io/documentation/pull/181/commits/33bc51f6ccc4fff05f2e568f2ea2e32df0ccf3fa and https://github.com/Exabyte-io/documentation/pull/181/commits/a9f3c596c34e18f68e34c2aaacb24c279f2473b2

It's worth noting that there's 154 existing instances of the word "you" in the documentation.